app/views/post_history/post.html.erb

```diff
@@ -4,10 +4,10 @@
   <%= render 'posts/type_agnostic', post: @post, show_category_tag: true, show_type_tag: true, last_activity: false %>
 </div>
 
-<% @history.each do |event| %>
+<% @history.each.with_index do |event,index| %>
   <details class="history-event">
     <summary>
-      <strong><%= event.post_history_type.name.humanize %></strong>
+      <strong>#<%= @history.size - index %>: <%= event.post_history_type.name.humanize %></strong>
```

Comment:
Just out of curiosity, why size - index instead of index +1?

Reply:
The indices need to be reversed to be in chronological order, hence size - index.

Reply:
Ah, of course